kontroller/qml/SettingsPage.qml

```diff
@@ -90,14 +90,13 @@ Page {
             }
 
             SectionHeader {
-                text: qsTr("Common")
+                text: qsTr("General")
```

Comment:
As in the menu?

Reply:
This is a relicate. Common is the idea that it was common to all servers. General is fine as well. In french i have put something that could translate as "global parameters".